tests/e2e/utils/command.go

```diff
@@ -4,19 +4,20 @@
 package utils
 
 import (
-	"context"
 	"fmt"
 	"strings"
 	"time"
 
+	"github.com/ethereum/go-ethereum/log"
 	"github.com/go-cmd/cmd"
 )
 
-func RunCommand(ctx context.Context, bin string, args ...string) (cmd.Status, error) {
-	Outf("{{green}}running '%s %s'{{/}}\n", bin, strings.Join(args, " "))
+// RunCommand starts the command [bin] with the given [args] and returns the command to the caller
+func RunCommand(bin string, args ...string) (*cmd.Cmd, error) {
```

Comment:
This can be optimized to stream the output more efficiently

Reply:
We can use `cmd.Options` and `cmd.NewCmdOptions` to do this via streaming rather than using the `Status()` call, which the documentation explains is the more efficient pattern.

Reply:
This can be a separate PR in the future if needed.